test/backend/test_node.py

```diff
@@ -1237,6 +1255,75 @@ def test_where(self):
     output = run_node(node_def, [c, x, y])
     np.testing.assert_almost_equal(output["Z"], np.where(c, x, y))
 
+  def test_scatter_nd(self):
```

Comment:
Please filter out the new tests so the target CI, currently for opset 10, would pass.